map2loop/project.py

```diff
@@ -351,27 +344,15 @@ def get_sampler(self, datatype: Datatype):
         """
         return self.samplers[datatype].sampler_label
 
-    @beartype.beartype
-    def set_minimum_fault_length(self, length: float):
-        """
-        Set the cutoff length for faults to ignore
-
-        Args:
-            length (float):
-                The cutoff length
-        """
-        self.deformation_history.set_minimum_fault_length(length)
-
-    @beartype.beartype
     def get_minimum_fault_length(self) -> float:
         """
         Get the cutoff length for faults
 
         Returns:
             float: The cutoff length
         """
-        return self.deformation_history.get_minimum_fault_length()
-
+        return self.map_data.minimum_fault_length
```

Comment:
remove? or return the config minimum fault length?

Reply:
removed. now a method in map_data (makes more sense)